client/config.go

```diff
@@ -32,15 +30,15 @@ func init() {
 
 // ConfigCmd returns a CLI command to interactively create a
 // Gaia CLI config file.
-func ConfigCmd() *cobra.Command {
+func ConfigCmd(defaultCLIHome string) *cobra.Command {
```

Comment:
Should really be `cliHome` as it doesn't have to "default", but not a huge deal.

Reply:
True that

Reply:
Well, actually 🤓the only place this variable is used is in setting the default below, so maybe `defaultCLIHome` is right? 🤷‍♂️